app/models/user.rb

```diff
@@ -64,6 +64,10 @@ def email_changed?
     false
   end
 
+  def holding?(stock)
+    portfolio&.portfolio_stocks&.exists?(stock_id: stock.id)
```

Comment:
It's showing holding true even if I sell all of my stocks 🤔 

If you check the PortfolioPosition, we make sure to show only stock that has `.having("SUM(portfolio_stocks.shares) > 0")`, so if I had 2, then I sold 2, then I don't have that stock anymore.